src/yb/yql/cql/cqlserver/cql_service.h

```diff
@@ -147,6 +152,10 @@ class CQLServiceImpl : public CQLServerServiceIf,
   // Tracker to measure and limit memory usage of prepared statements.
   MemTrackerPtr prepared_stmts_mem_tracker_;
 
+  // Password and hash cache.
+  boost::compute::detail::lru_cache<std::string, std::string> password_cache_;
```

Comment:
We have started to use thread safety annotations:
`GUARDED_BY(password_cache_mutex_);`